dockerfiles/Dockerfile.extensions

```diff
@@ -0,0 +1,29 @@
+# --------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------
+# Dockerfile to run ONNXRuntime and ONNXRuntime-Extensions from source from CPU only
+
+FROM ubuntu:18.04
+MAINTAINER Changming Sun "chasun@microsoft.com"
```

Comment:
Maintainer was deprecated. Use label instead.
```
LABEL maintainer="..."
```
Also shall we use group mail here?